README.md

```diff
@@ -64,7 +64,7 @@ This project is Leaderboard microvers which will be used for learning in the mic
 
 # LIVE DEMO 
 
-- [] coming soon
+- [click-here](#https://axcell-0.github.io/Leaderboard/)
```

Comment:
Nice job implementing the changes requested by the previous reviewer and deploying your project😉 But currently, the live demo link on your readme displays as text and not as a link. and I think it was a mistake while editing the link. Kindly do well to make the necessary change. ensure to confirm that the link is clickable before submitting, to avoid any form of blockers👍🏼.

<img width="927" alt="image" src="https://github.com/axcell-0/Leaderboard/assets/64643995/c60df1f8-a601-4474-a56f-939ce631e9a4">

Also, I noticed the link you were going to use doesn't show the actual project because the project contains dependencies from `webpack` and should be deployed in a different way. I will suggest you just take the live demo section from your readme since deployment isn't a requirement for this milestone or You can use a simple text under the section saying `Coming soon`👍🏼.